.github/actions/release/run

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+#
+# This script creates a nice markdown table with the build artifacts' shas, links to download
+# the Wasm modules, and links to the `sha256sum` steps in CI for verification.
+
+set -euo pipefail
+
+# This script sets "RELEASE_ACTION_BODY_FILE" to the path of the generated file. The script is run twice
+# (once when called with 'uses: ', once in post) meaning we can differentiate the first from second run
+# by reading "RELEASE_ACTION_BODY_FILE". On the "post" run, we just cleanup and exit.
+if [ -n "${RELEASE_ACTION_BODY_FILE:-}" ]
+then
+    >&2 echo "Cleaning up body file $RELEASE_ACTION_BODY_FILE"
+    if ! [ -f "$RELEASE_ACTION_BODY_FILE" ]
+    then
+        >&2 echo "strange, no file to remove"
+    else
+        rm "$RELEASE_ACTION_BODY_FILE"
+    fi
+    exit 0
+fi
+
+# The file where we'll progressively dump the release body content
+body=$(mktemp)
+>&2 echo "Using '$body' for body"
+
+# Start the body with a paragraph and table headers
+
+# NOTE: we link to the current release (not to master) because things might change
+if [ -z "${GITHUB_REF_NAME:-}" ]
+then
+    >&2 echo "No github ref name set"
+    rm "$body"
+    exit 1
+fi
+
+cat > "$body" <<EOF
+Read more in the [Build flavors](https://github.com/dfinity/internet-identity/tree/$GITHUB_REF_NAME#build-flavors) section of the README.
+| Filename | sha256 | Verify |
+| --- | --- | --- |
+EOF
+
+# Read all "INPUT_ASSETS" (where "ASSETS" is the input specified in action.yml)
+# https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs
+while IFS= read -r filename
+do
+    if [ -z "$filename" ]; then continue; fi
+    >&2 echo working on asset "$filename"
+
+    # Find out the Job ID
+    # XXX: Unfortunately GitHub actions doesn't give us a way to find out the Job ID explicitely.
```

Comment:
Should we add a comment somewhere near the matrix build definition, that links here and warns people that changes might break the release notes script?

Reply:
Done!